go-controller/pkg/ovn/egressfirewall.go

```diff
@@ -97,29 +99,39 @@ func newEgressFirewallRule(rawEgressFirewallRule egressfirewallapi.EgressFirewal
 
 // NOTE: Utilize the fact that we know that all egress firewall related setup must have a priority: types.MinimumReservedEgressFirewallPriority <= priority <= types.EgressFirewallStartPriority
 func (oc *Controller) syncEgressFirewall(egressFirewalls []interface{}) {
+	err := wait.PollImmediate(500*time.Millisecond, 60*time.Second, func() (bool, error) {
+		if err := oc.syncEgressFirewallRetriable(egressFirewalls); err != nil {
+			klog.Errorf("Failed (will retry) in egress firewall: %v", err)
```

Comment:
An error msg shall neither be capitalized.

Reply:
I think lint enforces the other way around: fmt.Errorf() should not be capitalized, but log messages do get capitalized... It looks like this PR passed lint though, so I think we're OK?

Reply:
This just got my attention but if lint is happy that's more than enough.